doc/nrf/known_issues.rst

```diff
@@ -504,6 +504,12 @@ Subsystems
 Bluetooth LE
 ============
 
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+NCSDK-9820: The :ref:`peripheral_lbs` - while holding one of the other buttons, the notification for button release is the same as for button press.
+
+  **Workaround:** Sample should filter which buttons were pressed before sending notification.
```

Comment:
Applications don't use samples, so no need to specify a workaround for the sample. Also modifying the code is not a workaround, it is the fix.

Reply:
Ok. I removed workaround section